client/galaxy/scripts/utils/uploadbox.js

```diff
@@ -6,110 +6,220 @@
     jQuery.event.props.push("dataTransfer");
 
     /**
-        Posts file data to the API
+        xhr request helper
     */
-    $.uploadpost = config => {
+    var _uploadrequest = config => {
+        var cnf = $.extend({
+            error_default: "Please make sure the file is available.",
+            error_server: "Upload request failed.",
+            error_login: "Uploads require you to log in.",
+            error_retry: "Waiting for server to resume...",
+        }, config);
+        console.debug(cnf);
```

Comment:
Leftover?

Reply:
I left in on purpose. Afaik we'll add a feature to the client builder to disable it for production.